.circleci/config.yml

```diff
@@ -36,26 +23,33 @@ commands:
     description: 'Installing dependencies'
     steps:
       - checkout
-      - restore_cache: *restore_yarn_cache
+      - restore_cache:
+          name: 'Restoring yarn cache'
+          keys:
+            - v1-yarn-cache-{{ checksum "yarn.lock" }}
+            - v1-yarn-cache
       - run: yarn install --pure-lockfile
-      - save_cache: *save_yarn_cache
-  build_artifacts:
-    description: 'Building artifacts'
+      - save_cache:
+          name: 'Saving yarn cache'
+          key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules/
+  restore_workspace:
+    description: 'Restoring workspace'
     steps:
-      - run: yarn build
+      - attach_workspace:
+          at: ~/project
   save_workspace:
+    description: 'Persisting workspace'
     steps:
       - persist_to_workspace:
-          root: *working_directory
+          root: ~/project
           paths:
-            - packages
-            - playground
-            - visual-testing-app
-            - application-templates
-  restore_workspace:
+            - merchant-center-application-kit
```

Comment:
Yarn [does the same thing](https://github.com/yarnpkg/yarn/blob/01713684fa5cc6b6d6143ac0bbea2c4f889d5fd8/.circleci/config.yml#L38-L41), let's see if it speed things up.

Reply:
Nice. Thanks. Nit: we could slap `~/project` into a `&project_directory`.